### ISSUE TITLE
failed installation

### ISSUE THREAD

**Issue by akhst7** [wilsonlabgroup/scMappR#3]
Hi, 

I am trying to update the package from githiub 
`scMappR (1a078adf0... -> 225e439b5...) [GitHub]`
but this fails with a warning as follows;

I```
nstalling package into ‘/Users/ahoji/Documents/R_lib_4’
(as ‘lib’ is unspecified)
* installing *source* package ‘scMappR’ ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error: object ‘markvario’ is not exported by 'namespace:spatstat'
Execution halted
ERROR: lazy loading failed for package ‘scMappR’
* removing ‘/Users/ahoji/Documents/R_lib_4/scMappR’
* restoring previous ‘/Users/ahoji/Documents/R_lib_4/scMappR’
Error: Failed to install 'scMappR' from GitHub:
```

I believe Spatstat have been updated recently, 

```
packageVersion("spatstat")
[1] ‘2.0.1’
```
and throwing a error.  

Thanks.

**Comment by DustinSokolowski** [comment]
Hey!

Thank you so much for your interest in the package! This error is being thrown because the spatstat package just got split into many packages. We rely on Seurat which uses spatstat and is currently adjusting their code to deal with the new spatstat (https://github.com/satijalab/seurat/issues/4226#issuecomment-800625766).

We have found that if you install Seurat from github prior to installing scMappR, then scMappR still installs well.

```
install.packages('remotes')
remotes::install_github(repo = 'satijalab/seurat', ref = 'develop')
install.packages("scMappR")
```

We have added these extra instructions to our readme and will update again once the new Seurat is on CRAN and installing goes back to normal.

Thank you again!
Dustin
